packages/contracts-bedrock/test/invariants/DelayedVetoable.t.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { StdUtils } from "forge-std/StdUtils.sol";
+import { StdInvariant } from "forge-std/StdInvariant.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { DelayedVetoable } from "../../src/universal/DelayedVetoable.sol";
+
+contract DelayedVetoable_Invariant_Harness is StdInvariant, Test {
+    event Forwarded(bytes data);
+
+    DelayedVetoable delayedVetoable;
+
+    // The address that delayedVetoable will call to
+    address dvTarget;
+
+    function setUp() public {
+        delayedVetoable = new DelayedVetoable({
+            target: address(dvTarget)
+        });
+
+        targetContract(address(delayedVetoable));
```

Comment:
As it stands this won't work. Proxies / call forwarders with invariant tests are kind of annoying - the fuzzer uses the contract's abi to crawl it, and the `DelayedVotable` contract only has a fallback. You'll probably need to target an actor a la:

```solidity
contract DelayedVetoableCaller {
    Vm internal immutable VM;

    DelayedVetoable public dv;
    bool public failed;

    constructor(Vm _vm) {
        VM = _vm;
    }

    function setDelayedVetoable(address _target) public {
        if (_target == address(0)) {
            _target = VM.addr(uint256(keccak256(abi.encode(_target))));
        }

        dv = new DelayedVetoable({ target: _target });
    }

    function doCall(bytes memory _toForward) external {
        if (address(dv) == address(0)) {
            setDelayedVetoable(VM.addr(uint256(keccak256(_toForward))));
        }

        bytes32 forwardHash = keccak256("Forwarded(bytes)");
        VM.expectCall(address(uint160(uint256(VM.load(address(dv), bytes32(0))))), _toForward);
        VM.expectEmit(true, false, false, true, address(dv));
        assembly {
            log1(add(_toForward, 0x20), mload(_toForward), forwardHash)
        }

        (bool success,) = address(dv).call(_toForward);
        if (!success) {
            failed = true;
        }
    }
}
```

and adjust the condition of the invariant test to:

```solidity
contract DelayedVetoable_Invariants is DelayedVetoable_Invariant_Harness {
    /// @custom:invariant Calls are always forwarded to the target
    function invariant_callIsForwarded_succeeds() public {
        assertFalse(actor.failed() || this.failed());
    }
}
```

Reply:
As we discussed offline, I've decided to remove these tests for now.
They may be added back later on, and I've saved a copy of them.